src/payment/agreement_payment_process.ts

```diff
@@ -282,14 +285,17 @@ export class AgreementPaymentProcess {
       await this.paymentModule.rejectInvoice(invoice, message);
       this.logger.warn(`Invoice rejected`, { reason: message });
     } catch (error) {
-      const message = getMessageFromApiError(error);
-      throw new GolemPaymentError(
-        `Unable to reject invoice ${invoice.id} ${message}`,
-        PaymentErrorCode.InvoiceRejectionFailed,
-        undefined,
-        invoice.provider,
-        error,
-      );
+      this.logger.warn(`Invoice rejected`, { reason: message });
+      // TODO: this endpoint is not implemented in Yagna, it always responds 501:NotImplemented.
```

Comment:
I think instead of here, we should comment out the actual call to yagna in the api adapter and move the warning to `payment.module.rejectInvoice`. Just so the behavior is the same no if it's the payment process trying to reject or the user manually

Reply:
It turns out that this is not implemented only for debitNote, I moved the warn to the module and commented out the call to the api